src/librustdoc/html/static/js/main.js

```diff
@@ -1813,7 +1811,7 @@ href="https://doc.rust-lang.org/${channel}/rustdoc/read-documentation/search.htm
     // from settings.js, which uses a separate function. It's done here because
     // the minimum sidebar size is rather uncomfortable, and it must pass
     // through that size when using the shrink-to-nothing gesture.
-    function hideSidebar() {
+    const hideSidebar = function hideSidebar() {
```

Comment:
```suggestion
const hideSidebar = function() {
```

Similarly for all other occurrences of this change.

I guess this is just drive-by cleanup?

Reply:
> I guess this is just drive-by cleanup?

No, it prevents the function definitions from being [hoisted](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) above the null check.